src/csharp/Microsoft.Spark/ML/Feature/StopWordsRemover.cs

```diff
@@ -0,0 +1,150 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+using Microsoft.Spark.Sql;
+using Microsoft.Spark.Sql.Types;
+
+namespace Microsoft.Spark.ML.Feature
+{
+    /// <summary>
+    /// A <see cref="StopWordsRemover"/> that removes stop words from column.
+    /// </summary>
+    public class StopWordsRemover : FeatureBase<StopWordsRemover>, IJvmObjectReferenceProvider
+    {
+        private static readonly string s_stopWordsRemoverClassName =
+            "org.apache.spark.ml.feature.StopWordsRemover";
+
+        /// <summary>
+        /// Create a <see cref="StopWordsRemover"/> without any parameters
+        /// </summary>
+        public StopWordsRemover() : base(s_stopWordsRemoverClassName)
+        {
+        }
+
+        /// <summary>
+        /// Create a <see cref="StopWordsRemover"/> with a UID that is used to give the
+        /// <see cref="StopWordsRemover"/> a unique ID
+        /// </summary>
+        /// <param name="uid">An immutable unique ID for the object and its derivatives.</param>
+        public StopWordsRemover(string uid) : base(s_stopWordsRemoverClassName, uid)
+        {
+        }
+
+        internal StopWordsRemover(JvmObjectReference jvmObject) : base(jvmObject)
+        {
+        }
+
+        /// <summary>
+        /// Sets the column that the <see cref="StopWordsRemover"/> should read from
+        /// </summary>
+        /// <param name="value">The name of the column to as the source</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetInputCol(string value) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setInputCol", value));
+
+        /// <summary>
+        /// Sets the column that the <see cref="StopWordsRemover"/> to save the result
```

Comment:
Not sure this is clear, maybe copy from something like Tokenizer:

/// The <see cref="Tokenizer"/> will create a new column in the DataFrame, this is the
/// name of the new column.